efficientdet/keras/efficientdet_arch_keras.py

```diff
@@ -812,7 +812,7 @@ def __init__(self, model_name=None, config=None, name=''):
     # Feature network.
     feat_sizes = utils.get_feat_sizes(config.image_size, config.max_level)
     self.resample_layers = {}  # additional resampling layers.
-    for level in range(config.min_level, config.max_level + 1):
+    for level in range(6, config.max_level + 1):
```

Comment:
Why?

Reply:
I am trying to use model.summry(), but it reports errors if some variables are not used in call function.
Do you  know some better solutions?

Reply:
I agree with this change.
It should be commented on how to get the value.